### DISCUSSION TITLE
posts/tutos/my-first-app/7-time-tracker/

### DISCUSSION THREAD

**Discussion by giscus** [Kapusch/blog-dotnet-maui#9] [Blog comments]
# posts/tutos/my-first-app/7-time-tracker/

Maintenant que les fondations de la page principale sont prêtes, on va pouvoir commencer à disposer tous les éléments de contrôle. Commençons avec le minutage !

https://www.mauicestclair.fr/posts/tutos/my-first-app/7-time-tracker/

**Comment by Mkkaba** [comment]
C’est fort 💪 🥰
